### PR TITLE
[Improve] flink on yarn e2e testcase

### PR DESCRIPTION
**Pull request by wolfboys**
## What changes were proposed in this pull request

Issue Number: close #xxx 

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (yes / no)

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -124,12 +124,12 @@ jobs:
...
-          - name: ApplicationsFlink117OnYarnWithFlinkSQLTest
-            class: org.apache.streampark.e2e.cases.ApplicationsFlink117OnYarnWithFlinkSQLTest
-          - name: ApplicationsFlink118OnYarnWithFlinkSQLTest
-            class: org.apache.streampark.e2e.cases.ApplicationsFlink118OnYarnWithFlinkSQLTest
+          - name: Flink116OnYarnTest
+            class: org.apache.streampark.e2e.cases.Flink116OnYarnTest
+          - name: Flink117OnYarnTest
+            class: org.apache.streampark.e2e.cases.Flink117OnYarnTest
+          - name: Flink118OnYarnTest
+            class: org.apache.streampark.e2e.cases.Flink118OnYarnTest
```
Comment (SbloodyS) [inline .github/workflows/e2e.yml]: Please avoid this unnessnary change.

Comment (wolfboys) [reply .github/workflows/e2e.yml]: > Please avoid this unnessnary change.

This test case includes both Flink on YARN application and YARN per-job modes, so I think the name "ApplicationsFlink116OnYarnWithFlinkSQLTest" is too long and inaccurate. How about "FlinkSQL116OnYarnTest"?

Comment (SbloodyS) [reply .github/workflows/e2e.yml]: I'm ok with it.

**Comment by SbloodyS** [comment]
LGTM

**Review by VampireAchao — APPROVED** [review]
LGTM

**Pull request merged**